typescript/lib/config.ts

```diff
@@ -70,6 +71,7 @@ export class AIConfigRuntime implements AIConfig {
   prompts: PromptWithOutputs[];
 
   filePath?: string;
+  callbackManager: CallbackManager = new CallbackManager([]);
```

Comment:
It seems strange to make it required and construct it by default vs just having it optional and optionally passed into the constructor

Reply:
updated to default logger